docs/authentication.md

```diff
@@ -40,6 +40,32 @@ Or if it's called from a class context:
 Person.find(id: 1234)
 ```
 
+### Basic authentication method
+
+Flexirest provides two methods for HTTP Basic Auth. The default method is `:url`:
+
+```ruby
+class Person < Flexirest::Base
+  basic_auth_method :url
+end
+# Includes the credentials in the URL:
+#   https://my_username:my_password@example.com/
+```
+
+But you might want to keep the credentials out of your logs and use the `:header` method:
+
+```ruby
+class Person < Flexirest::Base
+  basic_auth_method :header
+end
+# Clean URL:
+#   https://example.com/
+# with the base64 encoded credentials:
+#   Authorization: Basic YXBpOmViNjkzZWMtODI1MmMtZDYzMDEtMDJmZDAtZDBmYjctYzM0ODU=
+```
+
+You need to configure your model as such:
+
```

Comment:
The documentation seems to finish at `:` but feels like there should be something after it?

Reply:
Woops, that was from a previous attempt to write the doc. I'll delete this line.